### PR TITLE
Fix #304: Add check for hotfix label

### LINKED ISSUE #304: Oppiabot should assign release coordinator on addition of hotfix labels
Oppiabot should assign release coordinator team when a PR is labelled with `PR: Needs to be hotfixed`

### PR DESCRIPTION
**Pull request by ankita240796**
## Explanation
Fixes #304: Adds a check for when hotfix label is added so that release team is ping and the PR is brought to their attention.

## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [x] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: lib/checkPullRequestLabels.js [hunk 1]
```diff
@@ -340,6 +341,27 @@ module.exports.checkCriticalLabel = async function (context) {
...
+ *
+ * @param {import('probot').Context} context
+ */
+module.exports.checkHotfixLabel = async function (context) {
+  /**
+   * @type {import('probot').Octokit.IssuesGetLabelResponse} label
+   */
+  const label = context.payload.label;
+  if (label.name === HOTFIX_LABEL) {
+    var CommentParams = context.issue({
```
Comment (vojtechjelinek) [inline lib/checkPullRequestLabels.js:356]:
```suggestion
    var commentParams = context.issue({
```

?

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! One nit and on big picture comment.

File: lib/utils.js [hunk 2]
```diff
@@ -19,6 +19,7 @@
 const { 'default': Axios } = require('axios');
 
 const DATASTORE_LABEL = 'PR: Affects datastore layer';
+const HOTFIX_LABEL = 'PR: Needs to be hotfixed';
```
[the remark this answers is not in the document]
Comment (ankita240796) [reply lib/utils.js:22]: Sorry, I did not get this. Do you mean automatically add `PR: for current release` when the hotfix label is added?

[the remark this answers is not in the document]
Comment (ankita240796) [reply lib/utils.js:22]: Updated this as per our discussion.

**Review by vojtechjelinek — APPROVED** [review]
LGTM!

**Comment by ankita240796** [comment]
Hi @seanlip @vojtechjelinek can we please merge this one? Thanks!

**Pull request merged**